lib/agent.js

```diff
@@ -127,7 +123,7 @@ Agent.prototype._getStats = function () {
   const stats = {
     frameCache: frameCacheStats
   }
-  if (typeof this._transport._getStats === 'function') {
+  if (this._transport && typeof this._transport._getStats === 'function') {
```

Comment:
REVIEW NOTE: This is a guard against agent._getStats() being called before agent.start()